sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c

```diff
@@ -150,8 +155,28 @@ PRINT_CONFIG_VAR(OPTICFLOW_METHOD)
 #endif
 PRINT_CONFIG_VAR(OPTICFLOW_DEROTATION)
 
+#ifndef OPTICFLOW_DEROTATION
+#define OPTICFLOW_DEROTATION TRUE
+#endif
+PRINT_CONFIG_VAR(OPTICFLOW_DEROTATION)
+
+#ifndef OPTICFLOW_DEROTATION_CORRECTION_FACTOR_X
+#define OPTICFLOW_DEROTATION_CORRECTION_FACTOR_X 1.0
+#endif
+PRINT_CONFIG_VAR(OPTICFLOW_DEROTATION)
+
+#ifndef OPTICFLOW_DEROTATION_CORRECTION_FACTOR_Y
+#define OPTICFLOW_DEROTATION_CORRECTION_FACTOR_Y 1.0
+#endif
+PRINT_CONFIG_VAR(OPTICFLOW_DEROTATION)
+
 #ifndef OPTICFLOW_MEDIAN_FILTER
-#define OPTICFLOW_MEDIAN_FILTER TRUE
+#define OPTICFLOW_MEDIAN_FILTER FALSE
+#endif
+PRINT_CONFIG_VAR(OPTICFLOW_MEDIAN_FILTER)
+
+#ifndef OPTICFLOW_KALMAN_FILTER
+#define OPTICFLOW_KALMAN_FILTER TRUE
```

Comment:
Not by standard, that is why the ifndef statement set to false at the median filter and true with the kalman filter